programs/manifest/src/state/market.rs

```diff
@@ -708,6 +708,20 @@ impl<Fixed: DerefOrBorrowMut<MarketFixed>, Dynamic: DerefOrBorrowMut<[u8]>>
             })?;
 
             if did_fully_match_resting_order {
+                // Get paid for removing a global order.
+                if get_helper::<RBNode<RestingOrder>>(dynamic, current_order_index)
+                    .get_value()
+                    .get_order_type()
+                    == OrderType::Global
+                {
+                    let global_trade_accounts_opt: &Option<GlobalTradeAccounts> = if is_bid {
```

Comment:
AddOrderToMarketArgs should just have one GlobalTradeAccounts mapped - is_bid or not is known at that point in time

Reply:
could need both. need your side if you are resting an order. need other side if matching